AIPscan/Aggregator/templates/storage_service.html

```diff
@@ -49,7 +49,16 @@
       <tr>
         <td>{{ mets_fetch_job.download_start }}</td>
         <td>{% if mets_fetch_job.download_end %} {{ mets_fetch_job.download_end - mets_fetch_job.download_start }} {% endif %}</td>
-        <td>{{ mets_fetch_job.total_packages }}</td>
+        <td>{{ mets_fetch_job.total_packages }} packages
+          <i class="fa fa-info-circle fa-lg" id="{{ mets_fetch_job.id }}" style="margin-left: 5px;"></i><br/>
```

Comment:
For HTML5, the preferred line break tag is `<br>`, no trailing space/slash needed: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Reply:
![themoreyouknow](https://user-images.githubusercontent.com/672121/120840640-5b383380-c51f-11eb-8c7b-206a7458dc7b.jpeg)

Reply:
I learned basic HTML back in the 90's and this was considered very important (yet totally irrelevant in the end) practice.